src/types/Score.ts

```diff
@@ -127,43 +122,48 @@ export const getScoreColors = (
   value: ScoreValue | null,
   theme: Theme
 ): { background: string; border: string; text: string } => {
-  switch (value) {
-    case 2:
-      return {
-        background: theme.palette.TwClrBgSuccess as string,
-        border: theme.palette.TwClrBrdrSuccess as string,
-        text: theme.palette.TwClrTxtSuccess as string,
-      };
-    case 1:
-      return {
-        background: theme.palette.TwClrBgSuccess as string,
-        border: theme.palette.TwClrBrdrSuccess as string,
-        text: theme.palette.TwClrTxtSuccess as string,
-      };
-    case 0:
-      return {
-        background: theme.palette.TwClrBgInfo as string,
-        border: theme.palette.TwClrBrdrInfo as string,
-        text: theme.palette.TwClrTxtInfo as string,
-      };
-    case -1:
-      return {
-        background: theme.palette.TwClrBgWarning as string,
-        border: theme.palette.TwClrBrdrWarning as string,
-        text: theme.palette.TwClrTxtWarning as string,
-      };
-    case -2:
-      return {
-        background: theme.palette.TwClrBgDanger as string,
-        border: theme.palette.TwClrBrdrDanger as string,
-        text: theme.palette.TwClrTxtDanger as string,
-      };
-    default:
-      return {
-        background: theme.palette.TwClrBgInfo as string,
-        border: theme.palette.TwClrBrdrInfo as string,
-        text: theme.palette.TwClrTxtTertiary as string,
-      };
+  const defaultColors = {
+    background: theme.palette.TwClrBgInfo as string,
+    border: theme.palette.TwClrBrdrInfo as string,
+    text: theme.palette.TwClrTxtTertiary as string,
+  };
+
+  if (!value && value !== 0) {
+    return defaultColors;
+  }
+
+  if (1 <= value) {
+    return {
+      background: theme.palette.TwClrBgSuccess as string,
+      border: theme.palette.TwClrBrdrSuccess as string,
+      text: theme.palette.TwClrTxtSuccess as string,
+    };
+  } else if (0 <= value && value < 1) {
```

Comment:
Given how the Scoring UI works, is possible to enter a floating point value for a score?

Reply:
The average score comes back as a float and needs the same coloring. Although some recent feedback suggests it should be rounded to the nearest integer (instead of rounding up like I have here) so I'll put this back how it was tomorrow